rpc/documentation/api.md

```diff
@@ -945,6 +946,19 @@ transactions (and no mined transactions).
 
 ___
 
+#### `GetTicket`
+
+The `GetTicket` method queries the wallet for the provided ticket.
+
+**Request:** `GetTicketRequest`
+
+- `bytes ticket_hash`: The hex encoded hash of the ticket.
+
+**Response:** `GetTicketsResponse`
+
+Refer to [GetTickets](#gettickets) response documentation.
+___
+
```

Comment:
Document that error "NotFound" means the transaction isn't known by the wallet and "IlegalArgument" as the client sent a wrong hash